CHANGELOG.md

```diff
@@ -3,13 +3,19 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [2.1.0] - TBD
+### Added
+- add `metastore_events_sns_filter` map variable that accepts a valid sns filter
+### Removed
+- `database_filter` and `metastore_events_filter` variables
+
 ## [2.0.0] - 2019-10-04
 ### Added
 - SQS permissions policy.
 - tags to the lambda.
 
 ### Changed
-- converting the filter_policy to a template.
+- converting the filter\_policy to a template.
```

Comment:
I don't think we need to escape underscores in markdown do we? It renders fine in GH.